specs/vrp-openapi.yaml

```diff
@@ -174,8 +174,6 @@ paths:
         - $ref: "#/components/parameters/x-customer-user-agent"
 
       responses:
-        '200':
```

Comment:
As it is removed in new spec. 
Any way, this change won't us as we are not parsing 201OBDomesticVRPFundsConfirmationResponse directly, we are only parsing OBDomesticVRPConsentResponse for success response.